mono/mini/method-to-ir.c

```diff
@@ -6117,6 +6124,8 @@ inline_method (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *fsig,
 		if (cfg->verbose_level > 2)
 			printf ("INLINE END %s -> %s\n", mono_method_full_name (cfg->method, TRUE), mono_method_full_name (cmethod, TRUE));
 
+		g_assert (is_ok (&cfg->error));
```

Comment:
Use `mono_error_assert_ok`, it does the same but with a message that gives more info.